### PR TITLE
Fix: Table rows delete: split call with query and form-data

### PR DESCRIPTION
**Pull request by martinjunger**
Jira: TER-87
Connection: https://github.com/keboola/connection/pull/4158

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3777,11 +3777,41 @@ This request is [asynchronous](#introduction/synchronous-and-asynchronous-calls)
...
++ Request using whereFilters, send as form-data (application/x-www-form-urlencoded)
+    + Headers
+
+            X-StorageApi-Token: your_token
+
+    + Body
+
+            whereFilters%5B0%5D%5Bcolumn%5D=id&whereFilters%5B0%5D%5Boperator%5D=eq&whereFilters%5B0%5D%5Bvalues%5D%5B0%5D=12344
+
++ Request using whereFilters, send as JSON (application/json)
```
Comment (tomasfejfar) [inline apiary.apib:3789]: Ten JSON nemáme nikde otestovaný a ani klient to tak neposílá zatím, ne?

**Review by tomasfejfar — CHANGES_REQUESTED** [review]
Ten JSON má mít test nebo nebýt v Apiary

File: apiary.apib (hunk 1 above)
Comment (martinjunger) [reply apiary.apib:3789]: jo, ten jsem chtel jeste odstranit, dik za pripomenuti

Comment (martinjunger) [reply apiary.apib]: odstraneno

**Comment by martinjunger** [comment]
ℹ️ rebase

**Review by tomasfejfar — APPROVED** [review]
Testy prošly, apiary upravené

**Pull request merged**